core/README.md

```diff
@@ -0,0 +1,160 @@
+# `@commercetools-test-data/core`
+
+This package provides the core functions to define the data models.
+
+## Install
+
+```bash
+$ yarn add -D @commercetools-test-data/core
+```
+
+## Creating a new data model
+
+Models are treated packages. Each model will be published as a standalone NPM package.
+
+To create a new model, add a new package in the `models` workspace with the following structure:
+
+```
+models/<package_name>
+├── LICENSE
+├── README.md
+├── index.ts
+├── package.json
+├── src
```

Comment:
What about `constants.ts` file that contains the constants?

Reply:
The structure here is mostly for the minimal setup. If certain models require more things, they can just do that.

Reply:
Except that `constants` was an established convention, a unified way of defining constants in test-data. Maybe it's worth using the same convention here as well, to avoid future discussions.

Reply:
Does it mean that every model should have a `constants` file?

Reply:
As far as I understood, yes, it behaves as a placeholder if there are no constants, just like the `presets` folder.